selftests/functional/test_sysinfo.py

```diff
@@ -116,9 +116,15 @@ def run_sysinfo_interrupted(self, sleep, timeout, exp_duration):
                                  "existing location '%s' contains:\n%s"
                                  % (sleep_log, path, os.listdir(path)))
 
+    @unittest.skipIf(int(os.environ.get("AVOCADO_CHECK_LEVEL", 0)) < 2,
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
     def test_sysinfo_interrupted(self):
         self.run_sysinfo_interrupted(10, 1, 15)
 
+    @unittest.skipIf(int(os.environ.get("AVOCADO_CHECK_LEVEL", 0)) < 2,
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
So is it too long or resource intensive?

Reply:
(I mean it doesn't make sense to me to say it's that or that, you skip it because of some reason and it'd be nice to mention the reason directly and not just some generic phrase)